### PR TITLE
Consider SCROLL_AMOUNT into the mouse wheel scroll condition

### LINKED ISSUE #25: Can't reach the end of the page
When using a mouse and scrolling (fast), you can't reach the end of the page. This issue is probably caused by the following `if` statement:
https://github.com/oberonamsterdam/horizontal/blob/bf5e888dbc6b7e1c3a1d2e5f0857e06d7440d789/src/index.ts#L179

When the distance is superior to the page/container width the scrolling action is dismissed, where it **should** be taken into account. To fix the issue, an `else` statement can be added to simply set the scroll to the maximum value (100% scroll).

I'm not opening a PR because I don't know how to do that, but since it seems easy to fix I hope this issue will be fixed quickly 😁.

### PR DESCRIPTION
**Pull request by maximedaoust**
Fixes #25

### CODE REVIEW

**Comment by arguiot** [comment]
I've tested the changes myself and it works very well. I hope this PR will be merged very soon

File: src/index.ts [hunk 1]
```diff
@@ -176,7 +176,7 @@ export default class HorizontalScroll extends EventEmitter {
         }
         const distance = Math.max(this.container.scrollLeft + offset, 0);
 
-        if (distance < this.container.scrollWidth - this.container.clientWidth) {
+        if (distance - SCROLL_AMOUNT < this.container.scrollWidth - this.container.clientWidth) {
```
Comment (matthijsoberon) [inline src/index.ts:179]: Hey, thanks for contributing! Works fine, but this should be changed to the `scrollAmount` that's passed to the constructor so it takes the actual scrollAmount that's configured. Which is by default equal to the `SCROLL_AMOUNT` variable, but this makes it more dynamic.

**Review by matthijsoberon — CHANGES_REQUESTED** [review]
Thanks for contributing, good job :). I've given some feedback on your PR, will merge it after you've updated it!

File: src/index.ts (hunk 1 above)
Comment (matthijsoberon) [reply src/index.ts:179]: Resolved in PR #32

**Comment by matthijsoberon** [comment]
Duplicate of #32

**Pull request closed without merging**